SPECS/php/php.spec

```diff
@@ -697,7 +697,7 @@ For PHP, FFI opens a way to write PHP extensions and bindings to C libraries
 in pure PHP.
 
 %prep
-%setup -q
+%setup -q -n php-%{version}
```

Comment:
I might be confusing some build details here, but I think the value you've provided for the `-n` switch is the default. It's not wrong, just might be redundant. Why is it needed?